File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/ContactFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.std.Contact;

public class ContactFilterValidator extends FilterValidator<Contact> {

    public ContactFilterValidator(CustomValidator<Contact> validator) {
        super(Contact.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/ContactFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.Contact
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class ContactFilterValidator(validator: Validator<Contact>) : FilterValidator<Contact>(
    Contact::class, validator)
